wordpress-mobile/WordPress-iOS
# Draft Extension: Update display string for Jetpack's draft action

Part of #19361 

## Description
Updates the display string for Jetpack's draft action from "Save as Draft" to "Save to Jetpack"

## Testing Instructions

1. Install Jetpack.
2. Go to the system's Photos app.
3. Select a photo.
4. Tap on the share button.
5. Scroll down.
6. Make sure the action reads "Save to Jetpack"

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

6. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
